tests/post/steps/test_ui.py

```diff
@@ -0,0 +1,43 @@
+import json
+
+from pytest_bdd import scenario, then
+import requests
+
+
+# Scenarios
+@scenario('../features/ui_alive.feature', 'Reach the UI')
+def test_ui(host):
+    pass
+
+
+@then("we can reach the UI")
+def reach_UI(host, version, request):
+    with host.sudo():
+
+        bootstrap_ip = request.config.getoption("--bootstrap-ip")
```

Comment:
This will not work, sadly. The MetalK8s UI container is only binding to the workload-plane IP, while this option is another "public" IP of the bootstrap node (neither control- nor workload-plane) that the test runner can see. We really need the test runner to have access to these networks.

It will be hard to find a valid workaround, so I'd suggest you skip the test if in multi-nodes context:

```python
if bootstrap_ip is not None:
    pytest.skip("Cannot yet run this test on multi-nodes deployment.")
else:
    ...
```

Reply:
See #889

Reply:
Agree, let's skip it for now.

Reply:
👌